wgsl/index.bs

```diff
@@ -132,7 +132,8 @@ TODO: What indicates the end of a line?  (E.g. A line ends at the next linefeed
   <thead>
     <tr><th>Token<th>Definition
   </thead>
-  <tr><td>`FLOAT_LITERAL`<td>`-?(([0-9]*.[0-9]+ | [0-9]+.[0-9]*((e|E)(+|-)?[0-9]+)?) | 0x([0-9a-fA-F]*.?[0-9a-fA-F]+ | [0-9a-fA-F]+.[0-9a-fA-F]*)(p|P)(+|-)?[0-9]+)`
+  <tr><td>`DECIMAL_FLOAT_LITERAL`<td>`(-?[0-9]*.[0-9]+ | -?[0-9]+.[0-9]*)((e|E)(+|-)?[0-9]+)?`
+  <tr><td>`HEX_FLOAT_LITERAL`<td>`-?0x([0-9a-fA-F]*.?[0-9a-fA-F]+ | [0-9a-fA-F]+.[0-9a-fA-F]*)(p|P)(+|-)?[0-9]+`
```

Comment:
Could we add a `<tr><td>`FLOAT_LITERAL`<td>DECIMAL_FLOAT_LITERAL | HEX_FLOAT_LITERAL and then leave the usages below?